include/z64scene.h

```diff
@@ -534,119 +534,119 @@ typedef union {
     /* 0x05 */ SCENE_UNSET_5,
     /* 0x06 */ SCENE_UNSET_6,
-    /* 0x07 */ SCENE_KAKUSIANA,
-    /* 0x08 */ SCENE_SPOT00,
+    /* 0x07 */ SCENE_KAKUSIANA, // Lone Peak Shrine & Grottos
+    /* 0x08 */ SCENE_SPOT00, // Cutscene Scene
     /* 0x09 */ SCENE_UNSET_9,
-    /* 0x0A */ SCENE_WITCH_SHOP,
-    /* 0x0B */ SCENE_LAST_BS,
-    /* 0x0C */ SCENE_HAKASHITA,
-    /* 0x0D */ SCENE_AYASHIISHOP,
+    /* 0x0A */ SCENE_WITCH_SHOP, // Magic Hags' Potion Shop
+    /* 0x0B */ SCENE_LAST_BS, // Majora's Lair
+    /* 0x0C */ SCENE_HAKASHITA, // Beneath the Graveyard
+    /* 0x0D */ SCENE_AYASHIISHOP, // Curiosity Shop
     /* 0x0E */ SCENE_UNSET_E,
     /* 0x0F */ SCENE_UNSET_F,
-    /* 0x10 */ SCENE_OMOYA,
-    /* 0x11 */ SCENE_BOWLING,
-    /* 0x12 */ SCENE_SONCHONOIE,
-    /* 0x13 */ SCENE_IKANA,
-    /* 0x14 */ SCENE_KAIZOKU,
-    /* 0x15 */ SCENE_MILK_BAR,
-    /* 0x16 */ SCENE_INISIE_N,
-    /* 0x17 */ SCENE_TAKARAYA,
-    /* 0x18 */ SCENE_INISIE_R,
-    /* 0x19 */ SCENE_OKUJOU,
-    /* 0x1A */ SCENE_OPENINGDAN,
-    /* 0x1B */ SCENE_MITURIN,
-    /* 0x1C */ SCENE_13HUBUKINOMITI,
-    /* 0x1D */ SCENE_CASTLE,
-    /* 0x1E */ SCENE_DEKUTES,
-    /* 0x1F */ SCENE_MITURIN_BS,
-    /* 0x20 */ SCENE_SYATEKI_MIZU,
-    /* 0x21 */ SCENE_HAKUGIN,
-    /* 0x22 */ SCENE_ROMANYMAE,
-    /* 0x23 */ SCENE_PIRATE,
-    /* 0x24 */ SCENE_SYATEKI_MORI,
-    /* 0x25 */ SCENE_SINKAI,
-    /* 0x26 */ SCENE_YOUSEI_IZUMI,
-    /* 0x27 */ SCENE_KINSTA1,
-    /* 0x28 */ SCENE_KINDAN2,
-    /* 0x29 */ SCENE_TENMON_DAI,
-    /* 0x2A */ SCENE_LAST_DEKU,
-    /* 0x2B */ SCENE_22DEKUCITY,
-    /* 0x2C */ SCENE_KAJIYA,
-    /* 0x2D */ SCENE_00KEIKOKU,
-    /* 0x2E */ SCENE_POSTHOUSE,
-    /* 0x2F */ SCENE_LABO,
-    /* 0x30 */ SCENE_DANPEI2TEST,
+    /* 0x10 */ SCENE_OMOYA, // Mama's House & Barn
+    /* 0x11 */ SCENE_BOWLING, // Honey & Darling's Shop
+    /* 0x12 */ SCENE_SONCHONOIE, // The Mayor's Residence
+    /* 0x13 */ SCENE_IKANA, // Ikana Canyon
+    /* 0x14 */ SCENE_KAIZOKU, // Pirates' Fortress
+    /* 0x15 */ SCENE_MILK_BAR, // Milk Bar
+    /* 0x16 */ SCENE_INISIE_N, // Stone Tower Temple
+    /* 0x17 */ SCENE_TAKARAYA, // Treasure Chest Shop
+    /* 0x18 */ SCENE_INISIE_R, // Inverted Stone Tower Temple
+    /* 0x19 */ SCENE_OKUJOU, // Clock Tower Rooftop
+    /* 0x1A */ SCENE_OPENINGDAN, // Before Clock Town
+    /* 0x1B */ SCENE_MITURIN, // Woodfall Temple
+    /* 0x1C */ SCENE_13HUBUKINOMITI, // Path to Mountain Village
+    /* 0x1D */ SCENE_CASTLE, // Ancient Castle of Ikana
+    /* 0x1E */ SCENE_DEKUTES, // Deku Scrub Playground
+    /* 0x1F */ SCENE_MITURIN_BS, // Odolwa's Lair
+    /* 0x20 */ SCENE_SYATEKI_MIZU, // Town Shooting Gallery
+    /* 0x21 */ SCENE_HAKUGIN, // Snowhead Temple
+    /* 0x22 */ SCENE_ROMANYMAE, // Milk Road
+    /* 0x23 */ SCENE_PIRATE, // Pirates' Fortress Interior
+    /* 0x24 */ SCENE_SYATEKI_MORI, // Swamp Shooting Gallery
+    /* 0x25 */ SCENE_SINKAI, // Pinnacle Rock
+    /* 0x26 */ SCENE_YOUSEI_IZUMI, // Fairy's Fountain
```

Comment:
```suggestion
    /* 0x26 */ SCENE_YOUSEI_IZUMI, // Great Fairy's Fountain
```
This is _Great Fairy_ fountain, right? There is still one small regular fairy fountain in the well like OOT (not its own scene but some players might think there are others as a scene), but I feel like it should include great fairy in the name.

Reply:
Nope:
![image](https://user-images.githubusercontent.com/12245827/188238935-6725c89c-b863-4712-8747-7a485078b339.png)